src/Shared/Data/ServiceInsightsDbContext.cs

```diff
@@ -94,22 +92,6 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .HasColumnType("datetime")
                 .HasColumnName("RECORD_UPDATE_DATETIME");
             entity.Property(e => e.ScreeningId).HasColumnName("SCREENING_ID");
-
-            entity.HasOne(d => d.EndCode).WithMany(p => p.Episodes)
-                .HasForeignKey(d => d.EndCodeId)
-                .HasConstraintName("FK_EPISODE_END_CODE_LKP");
-
-            entity.HasOne(d => d.EpisodeType).WithMany(p => p.Episodes)
-                .HasForeignKey(d => d.EpisodeTypeId)
-                .HasConstraintName("FK_EPISODE_EPISODE_TYPE_LKP");
-
-            entity.HasOne(d => d.FinalActionCode).WithMany(p => p.Episodes)
-                .HasForeignKey(d => d.FinalActionCodeId)
-                .HasConstraintName("FK_EPISODE_FINAL_ACTION_CODE_LKP");
-
-            entity.HasOne(d => d.ReasonClosedCode).WithMany(p => p.Episodes)
-                .HasForeignKey(d => d.ReasonClosedCodeId)
-                .HasConstraintName("FK_EPISODE_REASON_CLOSED_CODE_LKP");
```

Comment:
Don't think we want to remove these. Had you created the FKs in your local db when you re-scaffolded with EF?